decent-dao/fractal-interface
# Create Proposal UI Redesign (Safe/Usul)

## Description

Updated Create Proposal page to match the next UX design documents.

## Notes

The pages for this branch do not match the designs until it is merged with `update/fractal-design-0.1`, but the components function correctly.
Additionally, the panel on the right side `Proposal Details` is not completed and needs to have live added added in a future PR. 
See included inline review comments

## Testing

- Navigate to an existing DAO [0xB87AD11E40F66137BD4cbB5f9d9438451000251B]. View all proposals.
- In the upper left of the screen choose `Create Proposal` to create a new proposal.
- Verify Accordion functions operate correctly for expand / collapse (click the Transaction number header)
- Verify Add and Remove transactions work correctly.
